packages/ra-ui-materialui/src/detail/Create.tsx

```diff
@@ -81,6 +82,7 @@ export const Create = <
             disableAuthentication={disableAuthentication}
             hasEdit={hasEdit}
             hasShow={hasShow}
+            loading={defaultLoading}
```

Comment:
`<Create>` should allow overriding the `loading` component. Same for the others.

Reply:
Stupid me